lib/ansible/module_utils/artifactory.py

```diff
@@ -77,7 +78,8 @@ def __init__(self, username=None, password=None, artifactory_url=None,
             self.headers["X-JFrog-Art-Api"] = auth_token
 
         if self.name:
-            self.working_url = '%s/%s' % (self.artifactory_url, self.name)
+            self.working_url = '%s/%s' % (self.artifactory_url,
+                                          urlparse.quote(self.name))
```

Comment:
instead of quoting it, would this work instead:  
`                                          self.name.replace(" ", "%20"))`

Reply:
I would rather use the proper lib to escape all characters instead of just spaces.